### PR TITLE
Added bower.json

### PR DESCRIPTION
**Pull request by Hypercubed**
If you like here is a bower.json.  You will also need to add a tag for 0.1.0.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "copycss",
+  "version": "0.1.0",
+  "homepage": "https://github.com/moagrius/copycss",
+  "authors": [
+    "Mike Dunn"
```
Comment (Hypercubed) [inline bower.json:6]: Change as you would like.

File: bower.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  ],
+  "description": "jQuery extension to retrieve or copy all styles from an HTML element.",
+  "main": "jquery.copycss.js",
```
Comment (Hypercubed) [inline bower.json:9]: This is the key, helps us easily include this component in other projects.

**Comment by moagrius** [comment]
sorry for the delay.  looks good, merging now

**Comment by Hypercubed** [comment]
Nice. Thank you.  This helps us use your plugin as a dependency.  You will also need to push a semver Git tag.

**Pull request merged**